pjsip/src/pjsip/sip_parser.c

```diff
@@ -445,7 +446,7 @@ static pj_status_t init_parser()
 
     status = pj_cis_dup(&pconst.pjsip_PROBE_USER_HOST_SPEC, &pconst.pjsip_ALNUM_SPEC);
     PJ_ASSERT_RETURN(status == PJ_SUCCESS, status);
-    pj_cis_add_str( &pconst.pjsip_PROBE_USER_HOST_SPEC, UNRESERVED ESCAPED USER_UNRESERVED ":");
+    pj_cis_add_str( &pconst.pjsip_PROBE_USER_HOST_SPEC, UNRESERVED ESCAPED USER_UNRESERVED ";" ":");
```

Comment:
What is `pjsip_PROBE_USER_HOST_SPEC`? Is this just something that should match everything in a URI after the `sip:`, and so needs to include ;?